src/libraries/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.Experimental.cs

```diff
@@ -433,6 +433,26 @@ public abstract partial class AdvSimd : System.Runtime.Intrinsics.Arm.ArmBase
         public static System.Runtime.Intrinsics.Vector64<sbyte> PopCount(System.Runtime.Intrinsics.Vector64<sbyte> value) { throw null; }
         public static System.Runtime.Intrinsics.Vector64<double> SqrtScalar(System.Runtime.Intrinsics.Vector64<double> value) { throw null; }
         public static System.Runtime.Intrinsics.Vector64<float> SqrtScalar(System.Runtime.Intrinsics.Vector64<float> value) { throw null; }
+        public unsafe static void Store(byte* address, System.Runtime.Intrinsics.Vector128<byte> source) { }
```

Comment:
Were these auto-generated or manually added?

Reply:
Auto-generated with Microsoft.DotNet.GenAPI.dll 5.0.0-beta.20153.1

Hmm, why are they missing `throw null`?

Reply:
It's probably since functions return no value

Reply:
@tannergooding Are you concerned with the functions order?

Reply:
> Hmm, why are they missing throw null?

Yeah, that's why I was asking. @ericstj or @safern should be able to comment if this is expected.

Reply:
What do you mean the order? I've typically just gone with however GenAPI orders them.

Reply:
Yes that is expected. Throw null is to avoid compiler errors on method that doesn't return. Since this is a void method, we don't need to add throw null to its body.

Reply:
> 
> 
> What do you mean the order? I've typically just gone with however GenAPI orders them.

I would expect that all `Vector128` overloads go after `Vector64`, but I don't think it matters

Reply:
I think that is just because of how normal string comparisons work, we need <https://github.com/dotnet/runtime/issues/13979> or similar for the sorting to treat `64 < 128` rather than `1 < 6`